### ISSUE TITLE
voron 0.2 r1 pro updating mistakes...

### ISSUE THREAD

**Issue by kramerjunior** [FYSETC/FYSETC-Voron-0.2-Pro#5]
Hello, i'm French i'm a new owner of a beauty Voron 0.2 r1 pro from AE...
When i finally finish my build i make the Great Update from the interface....
then.....
[moonraker(1).log](https://github.com/FYSETC/FYSETC-Voron-0.2-Pro/files/12458183/moonraker.1.log)
[klippy(1).log](https://github.com/FYSETC/FYSETC-Voron-0.2-Pro/files/12458184/klippy.1.log)
![screen voron 01](https://github.com/FYSETC/FYSETC-Voron-0.2-Pro/assets/143453313/4f2f43e8-0f51-40c7-b792-43f211c316b4)
![screen voron 02](https://github.com/FYSETC/FYSETC-Voron-0.2-Pro/assets/143453313/ad869f8b-ca98-422e-a8db-2b46c0cf822a)
![screen voron 03](https://github.com/FYSETC/FYSETC-Voron-0.2-Pro/assets/143453313/7e9a8824-5d3e-48e2-afd0-a569394126d5)
![screen voron 04](https://github.com/FYSETC/FYSETC-Voron-0.2-Pro/assets/143453313/4fac366a-7c47-4b93-a241-cd1241e3c0bb)
![screen voron 05](https://github.com/FYSETC/FYSETC-Voron-0.2-Pro/assets/143453313/a18b1cfc-37b2-41ea-81a2-1d7b17116288)

Could you help me to find an issue? 
thank you all.

**Comment by shoikan** [comment]
It appears the instructions are a bit incorrect. They clearly and explicitly state to use a 12MHz crystal which does not appear to work, however I found that if you use the 8MHz option it does work. YMMV of course.

**Comment by kramerjunior** [comment]
it's the same problem with 8mhz... do you have an other idea ???

**Comment by shoikan** [comment]
No, unfortunately no idea. On top of the issue with the wrong crystal speed, my whole board seems to have given up the ghost since it no longer accurately reports temperatures all of a sudden. Can't test more things for you anymore since it's no longer built into my voron and will soon make it's way into the trash-bin if Fysetc keeps ignoring their support email box. Thoroughly disappointed with this board. Waste of $75-125 that I'd rather have spent on a board that keeps working.

**Comment by runningsystemchanger** [comment]
Check the instructions here: https://github.com/FYSETC/Catalyst_Kit_V2.0
The CATALYST 2.0 has STM32F446. Not 401.
